planner/optimizer/pom.xml

```diff
@@ -18,27 +18,27 @@
 
 	<artifactId>optimizer</artifactId>
 
+
+
+	<packaging>pom</packaging>
+	<modules>
+		<module>optimizer-service</module>
+		<module>optimizer-core</module>
+	</modules>
 	<dependencies>
 		<dependency>
-			<groupId>eu.seaclouds-project</groupId>
-			<artifactId>toscamodel</artifactId>
-			<version>1.0.0</version>
+			<groupId>org.testng</groupId>
+			<artifactId>testng</artifactId>
 			<scope>test</scope>
 		</dependency>
-
-		<dependency>
-			<groupId>org.yaml</groupId>
-			<artifactId>snakeyaml</artifactId>
-		</dependency>
 		<dependency>
 			<groupId>org.slf4j</groupId>
 			<artifactId>slf4j-api</artifactId>
 		</dependency>
 		<dependency>
-			<groupId>org.testng</groupId>
-			<artifactId>testng</artifactId>
-			<scope>test</scope>
+			<groupId>org.slf4j</groupId>
```

Comment:
remove this dependency: see https://github.com/SeaCloudsEU/SeaCloudsPlatform/commit/872e6c5d3ff6f2eddc9e06dd4a6c6547390f5918